Formula/python.rb

```diff
@@ -35,17 +35,17 @@ class Python < Formula
   skip_clean "bin/easy_install3", "bin/easy_install-3.4", "bin/easy_install-3.5", "bin/easy_install-3.6", "bin/easy_install-3.7"
 
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/f4/d5/a6c19dcbcbc267aca376558797f036d9bcdff344c9f785fe7d0fe9a5f2a7/setuptools-41.4.0.zip"
```

Comment:
Please keep the `files.pythonhosted.org` URLs throughout (see https://github.com/Homebrew/homebrew-core/pull/2003 for when this was decided)

Reply:
I'm fine with following whatever the standard practice is, but I admit I don't understand it. Would `https://files.pythonhosted.org/packages/source/p/pip/pip-19.3.1.tar.gz` be allowable?